Makefile

```diff
@@ -216,6 +233,47 @@ docs/fips.inc: $(wildcard docs/fips/*) bin/mk-filelist
 	@rm -f $@
 	./bin/mk-filelist docs/fips fips/ '*' >$@
 
+######################################################################
+##
+##  Policy page building section
+##
+
+.PHONY: technical-policies
+technical-policies: $(TECHNICAL_POLICIES) bin/md-to-html5 Makefile
+	for x in $(TECHNICAL_POLICIES); do \
+		d=$$(dirname $$x); \
+		f=$$(basename $$x .md); \
+		cat "$$x" \
+			| sed -E -e 's!https?://github\.com/openssl/(general|technical)-policies/blob/master/policies/(.*)\.md!../\1/\2.html!' \
```

Comment:
Side note (not for this PR): This makes me wonder whether the original policies should actually be updated to reference the website directly rather than github.

Reply:
Nah.  If you look at them on github, you probably want to continue walking around on github